src/rebar.app.src

```diff
@@ -15,6 +15,11 @@
                   tools,
                   eunit,
                   common_test,
+                  dialyzer,
```

Comment:
Ignore this, rebar the application is never started and so adding this won't indirectly require `wx` even though we never use it.